src/phoebus_driver.cpp

```diff
@@ -87,6 +87,57 @@ TaskListStatus PhoebusDriver::Step() {
   return status;
 }
 
+// TODO(BRR) This is required for periodic BCs, unless the issue is radiation not being
+// included in ConvertBoundaryConditions
```

Comment:
This is only called on initialization -- if I don't call this, periodic BCs dont seem to update ghost zones between problem generation and the first timestep. It's something that came up when I switched from calling the ProblemGenerator over `interior` rather than `entire` zones. Maybe a parthenon issue? I haven't completely closed the loop on this. But again this is only before timestepping begins, comms arent serialized for radiation during time cycling.